Imports files to be used for drawing, puts data in double array.

dvizdos opened the pull request:
Data imports weird. Will fix. @Sampendyal start making the draw program, will have data working ASAP.

[NetBeansProjects.zip](https://github.com/CITIntroToProg1516-6/PurpleAmerica/files/238568/NetBeansProjects.zip)

Commit:
Imports files to be used for drawing

Sampendyal committed:
Added files via upload

Hey, it looks like I have to add a new file this way, but everything should work as expected.

LaurenSk8s committed:
Merge pull request #1 from Sampendyal/master

Added files via upload

LaurenSk8s committed:
Updated while loops

Creating new constructors and while loops to read through state point files and election files

LaurenSk8s committed:
Added Blank methods to help with compiling erros

These methods will be filled in later just blank to try to help get rid of some errors

LaurenSk8s committed:
added getColor method that will be used to set polygon color 

added a getColor method that will be used to set polygon colors based on the percentage of votes

LaurenSk8s committed:
Merge pull request #2 from LaurenSk8s/master

Updated while loops

LaurenSk8s committed:
Drew USA

Why is this change important?: Drawing the USA map is half of the project
Why did you do what you did?: I drew filled polygons then drew unfilled polygons over them to have the boarders.

LaurenSk8s committed:
Merge pull request #6 from LaurenSk8s/master

Drew USA

Sampendyal committed:
Outline of USA prototype and Coordinate storing class

This change allows to modularize the project. I did this so our program could be more efficient. Please note that they coordinate class may need some fixing to be able to work with Lauren's most updated class (in terms of the variable names and types).

LaurenSk8s committed:
Merge pull request #7 from Sampendyal/master

Outline of USA prototype and Coordinate storing class

dvizdos committed:
Creates GUI

This allows us to generate the GUI for people to search states in.

dvizdos committed:
Generates GUI

Creates GUI, search, and button for Purple America.

dvizdos committed:
Merge pull request #12 from dvizdos/master

Generates GUI

LaurenSk8s committed:
I added Alaska and Hawaii

Why is this change important?: I added my special feature which is to add Alaska and Hawaii. (Alaska has counties, but Hawaii doesn't yet.)
Why did you make the change? It was my special feature to add Alaska and Hawaii and it helped me start to understand how to add the counties.

LaurenSk8s committed:
Merge pull request #13 from LaurenSk8s/master

I added Alaska and Hawaii

Sampendyal committed:
Colors the states either blue or red or green

This change is important because I was able to get it so that it would color the states. 
I made this change in order to be able to complete the next part of our project. 

I added the coloring code to this. I will need to integrate this into the main class, and will do that once I know that it won't cause any problems to the program.

Sampendyal committed:
Merge pull request #14 from Sampendyal/master

Colors the states either blue or red or green

This change is important because I was able to get it so that it would color the states. 
I made this change in order to be able to complete the next part of our project.

I added the coloring code to this. I will need to integrate this into the main class, and will do that once I know that it won't cause any problems to the program.

LaurenSk8s committed:
Drew USA Counties and Created a Menu for the Program

Why is this change important?: We needed the counties drawn for all of the USA states and a menu for users to select State/Country, Country with Counties or No Counties, What State etc.
Why did you do what you did/why did you make the change?: It is important that the counties are drawn as that is a big factor in our grade.  It is also important that the user has some control over what he/she sees.
*******Note: I did this in the same file as the original DrawMap that only drew the USA states NOT counties.Do NOT merge or you will lose things! Please download separately, copy into another class and name this one "DrawMapCounties" or something.*************************

LaurenSk8s committed:
Drew USA Counties and Created a Menu for the Program

Why is this change important?: We needed the counties drawn for all of the USA states and a menu for users to select State/Country, Country with Counties or No Counties, What State etc.
Why did you do what you did/why did you make the change?: It is important that the counties are drawn as that is a big factor in our grade.  It is also important that the user has some control over what he/she sees.
*******Note: I did this in the same file as the original DrawMap that only drew the USA states NOT counties.Do NOT merge or you will lose things! Please download separately, copy into another class and name this one "DrawMapCounties" or something.*************************

dvizdos committed:
PURPLE!

The map is purple. America is great again. Will have the GUI working tonight. Remember guys, they take our computers tomorrow so make sure to have stuff worked out and extra functionality working and integrated!

dvizdos committed:
Should be purple

Hey guys, made some changes and for some reason all the values are read in as zero. Can you guys take a look at it and see if you can figure out why this is? I made a new color class and edited the DrawMapCounties. Thanks!

dvizdos committed:
Add files via upload

dvizdos committed:
Add files via upload

LaurenSk8s committed:
Merge pull request #19 from dvizdos/master

Purple

Sampendyal committed:
Src Files

This is important because we need to have the files in the right place. I made this change because we needed to have all the files in the right place so that when we do our final pull request, nothing will go wrong.

dvizdos committed:
GUI

Allows user to chose between drawing states or drawing states with counties. Adding drawing specific states next.

Sampendyal committed:
Merge pull request #20 from Sampendyal/master

Src Files

LaurenSk8s committed:
Final Project for Group

Did it through a zip file so there is no way it could overwrite my partners' committed files.  I know commits are factored into the final grade, and I want to make sure everyone gets credit for their commits.

Why is this change important?: It is our final version of our project.
Why did you make the change?: Needed to submit final project

LaurenSk8s commented:
@cmneville Please see last commit in this pull request I made for final version of project

LaurenSk8s committed:
Final Project for Group

I put it in a zip file so there is no way I could overwrite my partners' committed files.  I know that commits factor into each person's final grade for this project, and I want each member to get the credit he/she deserves for his/her number of commits.

Why is this change important?: It is the final version of our project and is due tonight.
Why did you make the change?: We needed a final version

LaurenSk8s committed:
Merge pull request #17 from LaurenSk8s/master

DON'T MERGE WITHOUT RENAMING Drew USA Counties and Created a Menu for the Program
